src/components/icon/icon.component.ts

```diff
@@ -41,6 +41,21 @@ class IconController implements IComponentController {
         }
     }
 
+    $onChanges(changes) {
+        const svg = (this.$element[0] as HTMLElement).firstElementChild;
+
+        // tslint:disable-next-line: curly
+        if (!svg) return;
```

Comment:
Our tslint was made for keep consistency between components codes. We disable tslint rules only at the last case 😃

Reply:
Fixed my friend :wink: